instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -326,14 +326,30 @@ def collect_custom_response_headers_attributes(response_headers):
     return attributes
 
 
-def _parse_status_code(resp_status):
+def parse_status_code(resp_status):
     status_code, _ = resp_status.split(" ", 1)
     try:
         return int(status_code)
     except ValueError:
         return None
 
 
+def parse_active_request_count_attrs(req_attrs):
```

Comment:
dont you need to remove it as you are using otel_wsgi.parse_active_request_count_attrs() ?

Reply:
This change is from wsgi file. I created a common function in wsgi file to be used in other wsgi-based frameworks.

Reply:
ok got it.